src/containers/Transactions/simpleTab.scss

```diff
@@ -66,6 +71,8 @@
         padding-left: 5px;
         color: $white;
         float: right;
+        font-size: 14px;
```

Comment:
@ckniffen FYI we really should have a font style library driven by design - do you agree?

Reply:
Its all over the place but yes.